blockchain/cfx.go

```diff
@@ -81,10 +84,10 @@ func (e cfxManager) GetTriggerJson() []byte {
 	switch e.p {
 	case subscriber.WS:
 		msg.Method = "cfx_subscribe"
-		msg.Params = json.RawMessage(`["logs",` + string(filterBytes) + `]`)
+		msg.Params = json.RawMessage(fmt.Sprintf("[%s,%s]", `"logs"`, string(filterBytes)))
```

Comment:
Since we're using %s with Sprintf, we shouldn't have to do the other string conversion
```suggestion
		msg.Params = json.RawMessage(fmt.Sprintf("[%s,%s]", `"logs"`, filterBytes))
```